lib/spack/docs/basic_usage.rst

```diff
@@ -39,27 +39,18 @@ Spack can install:
 
 .. command-output:: spack list
 
-The packages are listed by name in alphabetical order.  If you specify a
-pattern to match, it will follow this set of rules.  A pattern with no
-wildcards, ``*`` or ``?``, will be treated as though it started and ended with
-``*``, so ``util`` is equivalent to ``*util*``.  A pattern with no capital
-letters will be treated as case-insensitive. You can also add the ``-i`` flag
-to specify a case insensitive search, or ``-d`` to search the description of
+The packages are listed by name in alphabetical order.
+A pattern to match with no wildcards, ``*`` or ``?``,
+will be treated as though it started and ended with
+``*``, so ``util`` is equivalent to ``*util*``.  All patterns will be treated
+as case-insensitive. You can also add the ``-d`` to search the description of
 the package in addition to the name.  Some examples:
 
-All packages whose names contain "sql" case insensitive:
+All packages whose names contain "sql" :
```

Comment:
Can you remove the space before the colon to match the rest of the documentation? Other than that, looks good! I like how much simpler the new `spack list` is.